spec/features/visitor/get_results_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe "As a visitor" do
+  scenario "I can take a quiz and it will bring me to my recommendations" do
+    visit '/quiz'
+
+    click_link "See My Results"
+    expect(current_path).to eq('/results')
+  end
+end
```

Comment:
Test will be more robust once we know how the quiz will function, store data, and return games associated with keywords